test/db/asm/x86_64

```diff
@@ -54,10 +54,8 @@ a "mov rax, 0x100000000" 48b80000000001000000
 ad "movabs rax, 0x100000000" 48b80000000001000000
 ad "mov rax, 0" 48c7c000000000
 a "mov rax, -1" 48c7c0ffffffff
-adB "mov rax, 0xffffffffffffffff" 48c7c0ffffffff
-a "mov rax, -2" 48c7c0feffffff
```

Comment:
why removing this test? I'd keep it if it still works. If it doesn't mark it as broken.

Reply:
it was removed since -2 is not much different from -1 ... but keeping it is also fine 0257ab8b5241e8f17c990680145041b17ebf9ed1